lib/Field/FieldNew.js

```diff
@@ -0,0 +1,47 @@
+import React, { createContext } from 'react';
+import { bool, node, oneOf, string } from 'prop-types';
+import cx from 'classnames';
+import { FieldContent } from './FieldContent';
+import { FieldMeta } from './FieldMeta';
+import { FieldLabel } from './FieldLabel';
+import { FieldInstructions } from './FieldInstructions';
+
+const InStructureEditModeContext = createContext({
+  inStructureEditMode: false
+});
+
+export function FieldNew({ children, className, dir, inStructureEditMode }) {
+  return (
+    <InStructureEditModeContext.Provider value={{ inStructureEditMode }}>
+      <div
+        className={cx(
+          'grid grid-cols-12 gap-4 field field-new has-formatting',
+          className,
+          { field__editor: inStructureEditMode }
+        )}
+        dir={dir}
+      >
+        {children}
+      </div>
+    </InStructureEditModeContext.Provider>
+  );
+}
+
+FieldNew.propTypes = {
+  children: node.isRequired,
+  className: string,
+  dir: oneOf(['ltr', 'rtl']),
+  inStructureEditMode: bool
+};
+
+FieldNew.defaultProps = {
+  className: '',
+  dir: 'ltr',
+  inStructureEditMode: false
+};
+
+FieldNew.Content = FieldContent;
```

Comment:
💚What do you think to also adding `FieldActions`, `FieldValidations` here, as they are both also used to compose a field?